api/services/elastigroup/aws/schemas/elastigroup-target-scale.yaml

```diff
@@ -0,0 +1,46 @@
+type: object
+title: target scaling config
+properties:
+  source:
+    title: Source
+    description: "The source of the metric Supported values: “cloudWatch”, “spectrum”"
+    type: string
+    example: spectrum
+  policyName:
+    type: string
+    description: |
+      The policy name
+    example: "my-up-policy"
+  metricName:
+    type: string
+    description: |
+      The name of the metric.
+    default: "CPUUtilization"
+    example: "CPUUtilization"
+  statistic:
+    type: string
+    description: |
+      The metric statistics to return.
+    enum: [ average , sum , sampleCount , maximum , minimum , percentile ]
+    example: "average"
+  unit:
+    type: string
+    description: |
+      The unit for the alarm's associated metric.
+    enum: [ seconds , microseconds , milliseconds , bytes , kilobytes , megabytes , gigabytes , terabytes , bits , kilobits , megabits , gigabits , terabits , percent , count , bytes/second , kilobytes/second , megabytes/second , gigabytes/second , terabytes/second , bits/second , kilobits/second , megabits/second , gigabits/second , terabits/second , count/second , none ]
+    example: "percent"
+  namespace:
+    type: string
+    description: |
+      The namespace for the alarm's associated metric.
+    default: "AWS/EC2"
+  target:
+    type: integer
+    description: |
+      The target value for the group
+    example: 1
+  cooldown:
+    type: integer
+    description: |
+      The amount of time, in seconds, after a scaling activity completes before any further trigger-related scaling activities can start.
```

Comment:
"The amount of time, in seconds, after a scaling activity completes and before..."